Added a section to notify the customer to use CLI or SDK for configuring custom environments in BYOC Scenarios (#466)
MicrosoftDocs/azure-ai-docs: articles/machine-learning/how-to-deploy-custom-container.md

> [!IMPORTANT]

> > The `model_mount_path` value must be a valid absolute path in Linux (the OS of the container image).

Reviewer: ```suggestion
> The `model_mount_path` value must be a valid absolute path in Linux (in the guest OS of the container image).
```

> > [!IMPORTANT]
> > For BYOC scenarios, where a custom `model_mount_path` is to be configured on an online deployment, the [The inference_config parameter](#the-inference_config-parameter) parameter is also required to be set in the custom environment created for the model, for the environment to be recognized as a custom environment. For such scenarios, please use Azure CLI or Python SDK, as there are known limitations in configuring `inference_config` parameter when creating custom environment from Azure Portal.
> 

Reviewer 2: ```suggestion
> [!IMPORTANT]
> For BYOC scenarios, where a custom `model_mount_path` is to be configured on an online deployment, you must set the [`inference_config` parameter](#the-inference_config-parameter) in the custom environment created for the model, in order for the environment to be recognized as a custom environment. For such scenarios, use the Azure CLI or Python SDK to set the parameter. Do not try to set the parameter when creating the custom environment in the Azure portal, as there are some known limitations with configuring the `inference_config` parameter this way.

```

> > For BYOC scenarios, where a custom `model_mount_path` is to be configured on an online deployment, you must set the [`inference_config` parameter](#the-inference_config-parameter) in the custom environment created for the model, in order for the environment to be recognized as a custom environment. For such scenarios, use the Azure CLI or Python SDK to set the parameter. Do not try to set the parameter when creating the custom environment in the Azure portal, as there are some known limitations with configuring the `inference_config` parameter this way.

Reviewer: ```suggestion
> `model_mount_path` is usable only in BYOC (Bring your own container) scenario. In BYOC scenario, the environment that the online deployment uses must have [`inference_config` parameter](#the-inference_config-parameter) configured. You can use Azure ML CLI or Python SDK to specify `inference_config` parameter when creating the environment. Studio UI currently doesn't support specifying this parameter.
```
